server/eventlog.go

```diff
@@ -425,6 +425,11 @@ func getGrubState(hash crypto.Hash, events []*pb.Event) (*pb.GrubState, error) {
 			continue
 		}
 
+		// skip parsing EV_EVENT_TAG event since it likely comes from Linux.
+		if event.GetUntrustedType() == EventTag {
```

Comment:
I'm still little bit doubt to skip an event based on the event type. As the event type is unverified and thus untrusted. An attack may try to change a IPL event type to a EvenTag event type to skip the grub state check.

So I think we should also verify the digest and content of this event here (assuming it is possible), and then we can skip the event.